### DISCUSSION TITLE
Mountainous routes

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#547] [Q&A] [answered]
The BRouter Web Client has bike profiles where you can select “Consider Elevation”.
Wouldn't something like this also be an additional option for winding roads?
In mountainous regions it is more fun (at least for me) to ride a motorcycle...

**Comment by devemux86** [comment, marked as answer]
This option is also available in Cruiser for all bike and foot profiles.

Have you seen how it is used in BRouter rules?
Then, if possible, we could also adapt this to the driving profiles.
